pages/02.install/04.macos/default.md

```diff
@@ -5,34 +5,35 @@ title: macOS
 ### Introduction
 ! No one on the dev team owns a Mac. These instructions are suggested by our users. If you think you can improve them, please [edit this wiki](https://github.com/Kareadita/Wiki/edit/main/pages/02.install/04.macos/default.md) and submit a PR.
 
-1. [Download Kavita for macOS](https://www.kavitareader.com/#downloads-v1-macos)
+1. [Download Kavita for macOS](https://www.kavitareader.com/#downloads-v1-macos). The file you want is "kavita-osx-x64.tar.gz".
 
-2. Unpack Kavita to your User Folder or folder of choice.
+2. Unpack Kavita to your User Folder or folder of choice. For example, in Terminal:
+    * `cd ~`
+    * `mv ~/Downloads/kavita-osx-x64.tar.gz ~`
+    * `tar -xf kavita-osx-x64.tar.gz`
 
 3. Make Kavita executable.
-    * In Terminal: `sudo chmod +x ~/Kavita/Kavita`
+    * `sudo chmod +x ~/Kavita/Kavita`
 
 4. Disable Gatekeeper for Kavita's folder so that Kavita and all .dll and .dylib files can be opened
-    * In Terminal: `xattr -r -d com.apple.quarantine ~/Kavita/`
+    * `xattr -r -d com.apple.quarantine ~/Kavita/`
 
 5. Help Kavita find sqlite
-    * In Terminal: `export alias sqlite=sqlite3`
+    * `export alias sqlite=sqlite3`
 
-6. Start Kavita in Terminal. Need to keep this window open (minimised). 
-    * ````
-      cd ~/Kavita
-      nohup ./Kavita &
-      cat nohup.out
-      ````  
+6. Start Kavita in Terminal. You need to launch Kavita from within the `~/Kavita` folder, or else `appsettings.json` will fail to be acquired.
+    * `cd ~/Kavita`
+    * `./Kavita &`
 
 7. Open a browser window, at address: http://localhost:5000/
 
 ### Troubleshooting
-8. Change the port number if there's a clash, e.g. this error reported to the Terminal or log file (`config/logs/kavita.log`):
-   `Unhandled exception. System.IO.IOException: Failed to bind to address http://[::]:5000: address already in use.`
-    * Edit the file `config/appsettings.json` and change the value near the end to 
-      `"Port": 5555`
-9. Open a browser window, at address: http://localhost:5555/
+8. If you see an error like this, you have a Port clash. <sub><i>`Unhandled exception. System.IO.IOException: Failed to bind to address http://[::]:5000: address already in use.`</i></sub>
```

Comment:
These shouldn't be steps continuing from previous section as they are optional only if conflict. Starting these back at 1 would be more ideal.

Reply:
done.